### DISCUSSION TITLE
[Azure Key Vault] Max number of keys extracted from AKV by the Console

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#531] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13.0.5

### Description

Hi guys,
this morning I successfully configured an Azure Key Vault on my Company on the Console and linked it to my project.

When I saved my project, the Console was successfully able to get secrets from AKV and to write the keys inside the `external-secret` file, but the console only got the first 25 keys from AKV.

Is it a Console limit? Can it be increased?

Thanks

### Actual Outcome

The console does not get all the keys

### Expected Outcome

The console gets all the keys

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! It is the default limit given by AKV. We should take all keys, so we need to fix this on Console side.

**Reply by frengooo** [reply]
Hi @davidebianchi, thanks for the reply.

I have another question, why the console takes all the keys? It's not possible to let the console get only the keys defined on the project?

**Reply by davidebianchi** [reply]
Not at the moment, the console does not know the keys used by the configuration. This would be a future improvement
